### ISSUE TITLE
This repo still uses `master` instead of `main`

### ISSUE THREAD

**Issue by mikeroswell** [dushoff/newMakestuff#1]

**Comment by dushoff** [comment]
Do you want to make a case why I should worry about this?

**Comment by mikeroswell** [comment]
Nope. Just extrapolating from installation notes on shellpipes   
  

  https://dushoff.github.io/shellpipes/
  
  
  
  
  
  

  
  
>

**Comment by dushoff** [comment]
I see. I just use whatever default github supplied. I could see an argument for changing to "main" specifically for a repo that we are suggesting people to fork, which I think this is.
